src/main/java/unrc/dose/UserStat.java

```diff
@@ -9,7 +9,39 @@
 */
 package unrc.dose;
 
+import org.javalite.activejdbc.Base;
 import org.javalite.activejdbc.Model;
+import org.javalite.activejdbc.DBException;
 
 public class UserStat extends Model {
+	
+	/**
+	 * Create the statistics for a given user.
+	 * @param u The user who statistics we will create. 
+	 */
+	public static void createUserStat(User u){
+		UserStat stat = new UserStat();
+		stat.set("user_id",u.get("id"));
+		stat.set("created_challenges", 0);
+		stat.set("solved_challenges",0);
+		stat.set("current_points", 0);
```

Comment:
Instead of using methods `#set(String, Object)` and `#get(String)` which returns or takes an Object, you should use type specific ones, for example: `#setInteger(String, Object)` and `#getInteger(String)`. Using Type specific methods will improve code readability and, in case of getters, avoid having to having explicit casts.